recipe/meta.yaml

```diff
@@ -19,21 +19,21 @@ source:
     - h5py_win_setup_build.py.patch
 
 build:
-  number: 2
+  number: 3
 
 requirements:
   build:
     - python
     - numpy x.x
-    - hdf5
+    - hdf5 1.8*
     - cython
     - six
     - pkgconfig
 
   run:
     - python
     - numpy x.x
-    - hdf5
+    - hdf5 1.8*
```

Comment:
I can live with it.

When I added HDF5, I was like this is a patch release and we like newer versions of things it shouldn't cause any issues. Not to mention, I did not realize to the extent you guys depended on HDF5 and the extent to which your packages were supported by Continuum.

With 20-20 hindsight, it appears I have made an egregious error. So, yeah, let's bump the build number, but pin it to `1.8.15.1` exactly until we get more of your stack in and figure out dependency management issues here. Might want to move the HDF5 packages we have into a dev subchannel or something so they don't get pull down. That way we can end the nightmare that has happened to the HDF5 dependent stack.